e2pg/schema.sql

```diff
@@ -60,6 +60,28 @@ CREATE TABLE public.nft_transfers (
 );
 
 
+CREATE TABLE public.erc4337_userops (
+    contract bytea,
+    userOpHash bytea,
+    userOpSender bytea,
+    userOpPaymaster bytea,
+    userOpNonce numeric,
+    userOpSuccess boolean,
+    userOpActualGasCost numeric,
+    userOpActualGasUsed numeric,
```

Comment:
by pg convention you want all lowercase, something like

```
op_initcode
op_hash
op_sender
...
```

Reply:
done